File: codewars/dart/8kyu/apple_turnover.dart
import 'dart:math';

void main(List<String> args) {
  print(apple('50'));
  print(apple(2));
}

String apple(dynamic a) {
  if (a.runtimeType == String) {
    a = int.parse(a);
  }
  a = pow(a, 2);
  if (a > 1000) {
    return "It's hotter than the sun!!";
  }
  return 'Help yourself to a honeycomb Yorkie for the glovebox.';
}

File: codewars/python/8kyu/apple_turnover.py
def apple(x):
  x = int(x) ** 2
  if x > 1000:
    return "It's hotter than the sun!!"
  return "Help yourself to a honeycomb Yorkie for the glovebox."


if __name__ == '__main__':
  print(apple('50'))
  print(apple(4))
  print(apple('12'))
  print(apple(60))
